src/main/java/org/ehrbase/fhirbridge/ehr/converter/specific/virologischerBefund/PCRObservationConverter.java

```diff
@@ -0,0 +1,65 @@
+package org.ehrbase.fhirbridge.ehr.converter.specific.virologischerBefund;
+
+import ca.uhn.fhir.rest.server.exceptions.UnprocessableEntityException;
+import com.nedap.archie.rm.archetyped.FeederAudit;
+import com.nedap.archie.rm.datavalues.DvIdentifier;
+import org.ehrbase.fhirbridge.ehr.converter.generic.ObservationToObservationConverter;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.BefundObservation;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.LabortestBezeichnungDefiningCode;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.LabortestPanelCluster;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.NachweisDefiningCode;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.ProAnalytCluster;
+import org.ehrbase.fhirbridge.ehr.opt.geccovirologischerbefundcomposition.definition.VirusnachweistestDefiningCode;
+import org.hl7.fhir.r4.model.Observation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PCRObservationConverter extends ObservationToObservationConverter<BefundObservation> {
+
+    @Override
+    protected BefundObservation convertInternal(Observation resource) {
+        BefundObservation befundObservation = new BefundObservation();
+        befundObservation.setFeederAudit(createIdentifierSection(resource));
+        befundObservation.setLabortestBezeichnungDefiningCode(LabortestBezeichnungDefiningCode.DETECTION_OF_VIRUS_PROCEDURE);
+        befundObservation.setLabortestPanel(createLabortestPanel(resource));
+        return befundObservation;
+    }
+
+    private FeederAudit createIdentifierSection(Observation observation) {
+        FeederAudit feederAudit = new FeederAudit();
```

Comment:
delete feeder audit. This is atumatically done by the generic classes

Reply:
@SevKohler this is not the "normal" feederAudit stuff, it's part of the mapping.

Reply:
Well please ansure that this information is contained in the final Composition, as i see it its overwritten by the generic classes ?

Reply:
Just checked the JSON used to compare in tests. It contains the ID in FeederAudit as expected and documented.  
`` 
"feeder_audit" : {
      "_type" : "FEEDER_AUDIT",
      "originating_system_item_ids" : [ {
        "_type" : "DV_IDENTIFIER",
        "assigner" : "Organization/Charité",
        "id" : "94500-6_SARS-CoV-2-RNA-Presence-in-Respiratory-specimen",
        "type" : "org.hl7.fhir.r4.model.CodeableConcept"
      } ],
``

Reply:
So as far as i can see it everything is contained.